src/GraphQL.NewtonsoftJson/StringExtensions.cs

```diff
@@ -11,6 +12,8 @@ namespace GraphQL.NewtonsoftJson
     /// </summary>
     public static class StringExtensions
     {
+        private static readonly InputsConverter _inputsConverter = new();
```

Comment:
I'm not a fan of this new C#9 feature.

Reply:
Really? I think it’s pretty neat. Prevents the type from being specified twice. Sorta like var.

Reply:
I know but it looks a bit weird.

Reply:
This line does not exist anymore.